funnel/models/event.py

```diff
@@ -125,6 +134,24 @@ class Participant(BaseMixin, db.Model):
     __table_args__ = (db.UniqueConstraint('proposal_space_id', 'email'),)
 
     @classmethod
+    def get(cls, space, email, participant_attrs={}, create=False):
```

Comment:
`participant_attrs` is a mouthful. Call this `fields` or something light like that.